.gitignore

```diff
@@ -39,3 +40,4 @@ callgrind.out.*
 tests/autotests/autotests
 .ycm_extra_conf.py*
 *.stash
+
```

Comment:
GH is having some issues too... do a hard browser refresh to show it's been cleared. I spent over an hour with one editor on the Mac and it turned out it wasn't doing quotes in ISO standard e.g. the back and forward quotes. I really need to get OS X to Linux clipboard functions working... which means compiling the VNC client.